src/tests/functional/shared_test_classes/src/subgraph/matmul_multiply_fusion.cpp

```diff
@@ -46,6 +47,7 @@ void MatMulMultiplyFusion::SetUp() {
 
     auto transformed_function = function->clone();
     pass::Manager manager;
+    manager.register_pass<ov::pass::InitNodeInfo>();
     manager.register_pass<ov::pass::MatMulMultiplyFusion>();
```

Comment:
Currently, we must always call `InitNodeInfo`, which marks constant nodes, just before the target transformation in which `is_marked_as_constant_node` API is used. We must do that because it is not guaranteed that transformations between `InitNodeInfo` and target transformation don't make some nodes invalid (from `is_marked_as_constant_node` perspective).
Maybe it is better to always have a valid ConstantType state for all nodes in `ov::Model`? I see 2 ways to gain that:

1. Transfer constant graphs markup logic to `ov::pass::Validate`
2. Take care about ConstantType state during graph changes (e.g. output replacement)

What do you think?

Reply:
1. We should anyway to mark constants in initialization step to understand a node constant status in next iteration (transformation)
2. Agree, that we should update the related rt_info after call `graph_rewrite`